region/lowernorfair/east/Lower Norfair Fireflea Room.json

```diff
@@ -414,6 +500,24 @@
       ],
       "clearsObstacles": ["A"]
     },
+    {
+      "link": [2, 6],
+      "name": "G-Mode Morph Jump over the Fune",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphSpringBall"
+      ],
+      "devNote": [
+        "This is only useful if the item is Morph or a way to kill the Fune.",
+        "It may be possible to perform difficult blind Bomb jumps to remote acquire the item, but this would be incredibly difficult.",
```

Comment:
With Bombs, Springball, and hijump; the movements were very doable considering its blind.  Except PLMs were overloaded when I reached the item, and you don't have the health for spike hits in direct 😸